### PR TITLE
Fix duplicated textbox

### PR DESCRIPTION
**Pull request by nico-abram**
imagen

### CODE REVIEW

**Comment by nico-abram** [comment]
Fixed version in the deployment preview:

File: web/index.html [hunk 1]
```diff
@@ -203,7 +203,6 @@ <h1 id="title">The 2022 /r/place Atlas</h1>
 					<p>Please copy the text below and submit it as a<br>
 					new text post to <a target="_blank" href="https://www.reddit.com/r/placeAtlas2/">/r/placeAtlas2</a> on Reddit.</p>
 					<p>I will then check it and add it to the atlas.</p>
```
Comment (ProstoSanja) [inline web/index.html:205]: Please remove the duplicate text as well 
Also if you can, make <a> go directly to reddit and move it to the same div as button below

Comment (Hans5958) [reply web/index.html:205]: In my last PR, I moved it on the top so people won't miss it.

Yes, the dupe text shouldn't been needed (in fact just revert to the text I made excluding the warning)

Comment (nico-abram) [reply web/index.html]: Is this ok?
imagen

Comment (Hans5958) [reply web/index.html]: Last time I also add a text just so people won't get confused, but that is my minimum good.

Comment (nico-abram) [reply web/index.html]: Thanks for the warning, added back the same message

**Review by ProstoSanja — APPROVED** [review]

**Pull request merged**